llumnix/entrypoints/bladellm/eas_serve.py

```diff
@@ -0,0 +1,83 @@
+import signal
+import subprocess
+import sys
+import os
+import time
+import ray
+
+from llumnix.logging.logger import init_logger
+from llumnix.entrypoints.setup import connect_to_ray_cluster
+
+logger = init_logger("llumnix")
+
+connect_to_ray_cluster()
+
+
+@ray.remote
+class EasLaunchActor:
+    def __init__(self):
+        self.process = None
+
+    def launch_serve(self, command):
+        # avoid "RuntimeError: No CUDA GPUs are available"
+        os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+
+        logger.info("Launching command: {}".format(' '.join(command)))
+
+        try:
+            self.process = subprocess.Popen(
+                command,
+                preexec_fn=os.setsid
+            )
+            logger.info(f"Started serve launch command with PID {self.process.pid}")
+        except Exception as e:
+            logger.error(f"Subprocess error: {e}")
+        return self.process.pid
+
+    def stop(self):
+        #TODO(tongchenghao): not work
+        logger.info("Stopping subprocess...")
+        if self.process and self.process.poll() is None:
+            try:
+                logger.info(f"Sent SIGTERM to process group {os.getpgid(self.process.pid)}")
+                os.killpg(os.getpgid(self.process.pid), signal.SIGTERM)
+                self.process.wait(timeout=15)
+            except subprocess.TimeoutExpired:
+                logger.warning("Terminate timeout, using kill")
+                os.killpg(os.getpgid(self.process.pid), signal.SIGKILL)
+                self.process.wait()
+
+
+def find_gpu_node():
+    gpu_nodes = ray.nodes()
```

Comment:
gpu_nodes -> nodes here? because ray.nodes return all nodes no matter if it contains gpus.